src/utils/Processing.ts

```diff
@@ -564,6 +567,42 @@ export function writeFHIRResources(
   writeResources(outPackage.instances.filter(i => i._instanceMeta.usage !== 'Inline'));
 
   logger.info(`Exported ${count} FHIR resources as JSON.`);
+  return skippedResources;
+}
+
+export function writeFSHIndex(
+  outDir: string,
+  outPackage: Package,
+  inputDir: string,
+  skippedResources: string[] = []
+) {
+  const index: string[][] = [];
+  const otherIndex: any[] = [];
+  for (const [fileName, fshInfo] of outPackage.fshMap) {
+    if (!skippedResources.includes(fileName)) {
+      const relativeInput = path.relative(inputDir, fshInfo.file);
+      index.push([
+        fshInfo.fshName,
+        fshInfo.fshType,
+        relativeInput,
+        `${fshInfo.location.startLine} - ${fshInfo.location.endLine}`,
+        fileName
+      ]);
+      otherIndex.push({
+        fshFile: relativeInput,
+        fshName: fshInfo.fshName,
+        fshType: fshInfo.fshType,
+        startLine: fshInfo.location.startLine,
+        endLine: fshInfo.location.endLine,
+        outputFile: fileName
+      });
+    }
+  }
```

Comment:
I definitely like the idea of sorting. I'm torn on the sort order because it sounded like when this issue first came up that someone was looking to find the FSH file based on the JSON file, which would make your third suggestion most useful. But looking at the table and the contents, I'd prefer either of the other two.

All that to say, I like the idea, and I'm torn so I don't have a strong preference either.